CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+The signac team is committed to fostering a diverse and inclusive environment.
+The signac project has therefore adopted the [Contributor Covenant Code of Conduct](https://signac.io/conduct/).
+Please email [signac-support@umich.edu](mailto:signac-support@umich.edu) if you have any questions or concerns.
```

Comment:
Proposing the following - might be a little easier to read and slightly more friendly.

> # Code of Conduct
> The signac team is committed to fostering a welcoming community.
> Our Code of Conduct can be found here: https://signac.io/conduct/
> Feel free to contact [signac-support@umich.edu](mailto:signac-support@umich.edu) with any questions.
```

Reply:
I like the improved formatting, but I can't adopt your proposed changes since the [rails document](https://github.com/rails/rails/blob/master/CODE_OF_CONDUCT.md) is not in the public domain as far as I know.

Reply:
Good catch - I looked at a few examples but it looks like I pulled more from that version than I realized. Whatever you want to do here is fine.